clientconn.go

```diff
@@ -372,6 +372,18 @@ func (csm *connectivityStateManager) getNotifyChan() <-chan struct{} {
 	return csm.notifyChan
 }
 
+// InvokerStreamer defines the functions clients need to perform unary and
+// streaming RPCs.  It is implemented by ClientConn.
+//
+// This API is EXPERIMENTAL.
+type InvokerStreamer interface {
```

Comment:
A thought: if we also want to use this interface for interceptors, we need to make sure all ClientConn features are available.
Method `ClientConn() *ClientConn` should work.